oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -143,3 +146,31 @@ def map_boot(self, boot_list):
             redfish_boot_list.append('None')
 
         return redfish_boot_list
+
+    def _fill_resource_block_members(self, server_profile, drives):
+        resource_block_uuids = \
+            self._get_resource_block_uuids(server_profile, drives)
+
+        resource_blocks_base_uri = \
+            "/redfish/v1/CompositionService/ResourceBlocks/{}"
+        for resource_block_uuid in resource_block_uuids:
+            self.redfish["Links"]["ResourceBlocks"].append({
+                "@odata.id": resource_blocks_base_uri.format(
+                    resource_block_uuid
+                )
+            })
+
+    def _get_resource_block_uuids(self, server_profile, drives):
+        resource_block_uuids = list()
+        resource_block_uuids.append(self.server_hardware["uuid"])
```

Comment:
The server_hardware is the unique that is accessed by `self` reference, maybe could be better to keep the consistency remove it from `self` and pass it like the `server_profile` and `drives` objects (by parameters of method).